### ISSUE TITLE
some error in pryder

### ISSUE THREAD

**Issue by Tweetl** [spyder-ide/spyder#6400]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 890, in run_script
    self.execute_code(line, current_client, clear_variables)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole.py", line 936, in execute_code
    sw.execute(to_text_string(to_text_string(lines)))
  File "C:\ProgramData\Anaconda3\lib\site-packages\qtconsole\history_console_widget.py", line 47, in execute
    source, hidden, interactive)
  File "C:\ProgramData\Anaconda3\lib\site-packages\qtconsole\console_widget.py", line 632, in execute
    self._execute(source, hidden)
  File "C:\ProgramData\Anaconda3\lib\site-packages\qtconsole\frontend_widget.py", line 294, in _execute
    msg_id = self.kernel_client.execute(source, hidden)
AttributeError: 'NoneType' object has no attribute 'execute'

## Version and main components

* Spyder Version: 3.2.3 
* Python Version: 3.6.2
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.5.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.1 (OK)
sphinx >=0.6.6   :  1.6.3 (

**Comment by ccordoba12** [comment]
This is fixed in our latest release (3.2.6). Please update.

**Comment by CAM-Gerlach** [comment]
For the record, appears to be a duplicate of #5053 which was fixed in Spyder 3.2.4, released some time ago. You can update with ``conda update spyder``.
